docs/Config_Reference.md

```diff
@@ -1835,6 +1835,42 @@ z_offset:
 #   See the "probe" section for more information on the parameters above.
 ```
 
+### [load_cell_probe]
+
+Load-cell based probe. One may define this section (instead of a probe
+section) to enable a load-cell based probe. See
+[Load-cell probe guide](LoadCellProbe.md) and
+[command reference](G-Codes.md#load_cell_probe) for further
+information. Some of the parameters are determined in a semi-automatic
+[calibration procedure](LoadCellProbe.md#setup-for-new-printer-models).
+
+```
+[load_cell_probe]
+#
+# General parameters:
+#
+adc:
+#   ADC pin which digitizes the load cell signal.
+adc_rate:
+#   Rate (in samples per second) at which to request the ADC samples.
+max_abs_force:
+#   Maximum absolute value of measured force (in chosen unit), if exceeded
+#   printer will shutdown immediately.
+#
+# Parameters determined by semi-automatic procedure described in the
+# load-cell probe guide.
+#
+force_calibration:
+#   Conversion factor to get from ADC value to physical unit. Default value is
+#   1 (no conversion).
+stiffness:
+#   Stiffness/"spring constant", i.e. force per distance. Default value is a
+#   safe value resulting in very small step sizes.
```

Comment:
FWIW, I find it confusing to state there is some vague default.  Probably best to state explicitly what the default is.  I'd refrain from saying it is "safe" as we really have no idea if it could cause damage to hardware or not.